packages/common/core-interfaces/src/messages.ts

```diff
@@ -25,4 +25,9 @@ export interface ISignalEnvelope {
 		// eslint-disable-next-line @typescript-eslint/no-explicit-any
 		content: any;
 	};
+
+	/**
+	 * Client ID of the singular client the signal is being (or has been) sent to.
+	 */
+	targetClientId?: string;
```

Comment:
I guess there is no place to have this for v1 signals.
I commented elsewhere, but I think we start with only supporting targeted when service supports it.
When service does support v2, we don't care to double specify targetClientId.

Reply:
Removed targetClientId from SignalEnvelope. This was there for when we fallback on broadcast when no service support.